src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("요구사항 1. Set의 size() 메소드를 활용해 Set의 크기를 확인한다.")
+    void setSizeTest() {
+        assertThat(numbers.size())
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

크기를 비교하는 경우는 hasSize 사용을 추천드립니다.

Reply:
아하 넵 리뷰 감사합니다! :)